### ISSUE TITLE
Еxchange blocks to level

### ISSUE THREAD

**Issue by GIGABAIT93** [BentoBoxWorld/Level#142]
Add the ability to swap blocks to the island level. So players can exchange the island blocks they don't need and not build large cubes of blocks to increase the level

**Comment by tastybento** [comment]
Interesting idea.

**Comment by Poslovitch** [comment]
So I can turn my cobble into diamond blocks value? That sounds silly.

**Comment by tastybento** [comment]
No, I think he means you can bank them. Right now, to get the blocks to count they have to be on your island, so you see these islands with blocks of lapis or redstone just sitting there. 
I imagine that you could throw them into lava or something and they would shift your baseline island level up permanently.

**Comment by Poslovitch** [comment]
Oh, I get that. Sounds interesting.

**Comment by GIGABAIT93** [comment]
you can add the command /is level add (island_name) (vaule) - adds experience to the island.
and the exchange of blocks for experience everyone can do with the help of the GUI plugin
